rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -418,4 +420,18 @@ public Integer getMessageQueueMaxSize() {
     public boolean rpcZeroSignatureIfRemasc() {
         return configFromFiles.getBoolean("rpc.zeroSignatureIfRemasc");
     }
+
+    public double getTopBest() {
+        if (!configFromFiles.hasPath(PROPERTY_SYNC_TOP_BEST)) {
+            return 0.0D;
+        }
+
+        double value = configFromFiles.getDouble(PROPERTY_SYNC_TOP_BEST);
+
+        if (value >= 0.0D && value <= 100.0D) {
```

Comment:
probably 0 and 100 are legit values, wdyt?

Reply:
Addressed